migrations/frontend/1685525992_add_license_fields_to_support_auto_billing/up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE IF EXISTS product_licenses
+    ADD COLUMN IF NOT EXISTS site_id UUID NULL,
+    ADD COLUMN IF NOT EXISTS license_check_token bytea NULL,
+    ADD COLUMN IF NOT EXISTS revoked_at timestamptz NULL,
+    ADD COLUMN IF NOT EXISTS salesforce_sub_id text NULL,
+    ADD COLUMN IF NOT EXISTS salesforce_opp_id text NULL;
```

Comment:
nit: we may skip the default nullability declaration
```suggestion
    ADD COLUMN IF NOT EXISTS site_id UUID,
    ADD COLUMN IF NOT EXISTS license_check_token bytea,
    ADD COLUMN IF NOT EXISTS revoked_at timestamptz,
    ADD COLUMN IF NOT EXISTS salesforce_sub_id text,
    ADD COLUMN IF NOT EXISTS salesforce_opp_id text;
```

Reply:
True that, but it might be better to be explicit.